std_msgs/Header header
stfn_object_detector/Detection2D[] detections